src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -364,31 +398,33 @@ class CollapsibleColumns extends BasePlugin {
             const row = this.nestedHeadersPlugin.levelToRowCoords(parseInt(i, 10));
             const col = parseInt(j, 10);
 
-            this.markSectionAs(action === 'collapse' ? 'collapsed' : 'expanded', row, col, true);
-            this.toggleCollapsibleSection({
+            sectionToToggle.push({
               row,
               col
-            }, action);
-
+            });
           }
         });
       });
 
     } else {
-      objectEach(this.buttonEnabledList, (headerRow, i) => {
-        objectEach(headerRow, (header, j) => {
-          const rowIndex = parseInt(i, 10);
-          const columnIndex = parseInt(j, 10);
-
-          this.markSectionAs(action === 'collapse' ? 'collapsed' : 'expanded', rowIndex, columnIndex, true);
-          this.toggleCollapsibleSection({
-            row: rowIndex,
-            col: columnIndex
-          }, action);
-
+      arrayEach(this.buttonEnabledList, (headerRow) => {
```

Comment:
This code is a little confusing. The `if` expression is not necessary at all if you used argument destruction in the callback. This can be simplified to something like this?
```js
arrayEach(this.buttonEnabledList, ([headerRowIndex, columnIndexes]) => {
  arrayEach(columnIndexes, (columnIndex) => {
    const rowIndex = parseInt(headerRowIndex, 10);

    sectionToToggle.push({
      row: rowIndex,
      col: columnIndex
    });
  });
});
```

Reply:
Yes, You right. Thanks.